longest_palindrome_possible.java

```diff
@@ -0,0 +1,67 @@
+
+// BRUTE FORCE 
+// TC: O(n) checking through the entire hashmap [O(1) -search] for each character in the string and updating the values ,
+//          and 2 more iterations through hashmap for getting the even numbers and the checking for atleast one left over character or not
+// SC: O(n) for hashmap
+// Leetcode successful
+
+//DOUBTS:
+//1---> For iteration through Hashmpap, I used the entry set, do that take any space/memory?
+//2---> Is coming up with this kind of brute force approach good when I don't know the optimized soln?  [Any tips on how to improve better]
```

Comment:
Its quite normal to come up with brute force approach. Try to practice more problems so that you can easily identify the patterns.